torax/plotting/configs/default_plot_config.py

```diff
@@ -145,8 +145,8 @@
             suppress_zero_values=True,  # Do not plot all-zero data
         ),
         plotruns_lib.PlotProperties(
-            attrs=('q_brems', 'q_rad'),
-            labels=(r'$Q_\mathrm{brems}$', r'$Q_\mathrm{rad}$'),
+            attrs=('q_brems', 'q_imp'),
```

Comment:
nit: sorry for being so nitpicky, but while the source name is impurity_radiation_heat_sink , I think it's OK to maintain the shorthand q_rad, P_rad, Q_rad, which are all user-facing and standard convention.

We'll then have say q_rad, q_brems, q_cycl (coming soon) and this will be clear enough for fusion physicists

Reply:
Sure, no problem.